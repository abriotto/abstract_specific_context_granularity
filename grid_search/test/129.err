  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:11,  8.32it/s]  2%|###7                                                                                                                                                                                        | 2/99 [00:00<00:11,  8.11it/s]  4%|#######5                                                                                                                                                                                    | 4/99 [00:00<00:08, 10.78it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:06, 15.27it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:00<00:06, 14.45it/s] 11%|####################7                                                                                                                                                                      | 11/99 [00:00<00:05, 15.51it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:01<00:06, 12.94it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:01<00:07, 10.64it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:01<00:06, 12.22it/s] 19%|###################################8                                                                                                                                                       | 19/99 [00:01<00:06, 11.44it/s] 21%|#######################################6                                                                                                                                                   | 21/99 [00:01<00:06, 11.47it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:01<00:06, 11.08it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:06, 12.18it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:02<00:07,  9.89it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:02<00:07,  9.74it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:02<00:06,  9.73it/s] 33%|##############################################################3                                                                                                                            | 33/99 [00:03<00:07,  9.40it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:03<00:07,  8.70it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:08,  7.90it/s] 36%|####################################################################                                                                                                                       | 36/99 [00:03<00:08,  7.50it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:03<00:08,  7.01it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:03<00:09,  6.65it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:03<00:09,  6.57it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:04<00:09,  6.34it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:04<00:09,  6.35it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:08,  6.36it/s] 43%|#################################################################################2                                                                                                         | 43/99 [00:04<00:09,  6.21it/s] 44%|###################################################################################1                                                                                                       | 44/99 [00:04<00:09,  6.06it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:04<00:06,  8.20it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:05<00:07,  7.30it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:05<00:06,  7.74it/s] 51%|##############################################################################################4                                                                                            | 50/99 [00:05<00:06,  7.31it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:05<00:07,  6.78it/s] 53%|##################################################################################################2                                                                                        | 52/99 [00:05<00:07,  6.50it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:06<00:05,  7.69it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:06<00:06,  7.23it/s] 57%|#########################################################################################################7                                                                                 | 56/99 [00:06<00:06,  6.99it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:06<00:05,  7.51it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:05,  7.11it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:07<00:05,  7.58it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:07<00:05,  7.25it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:07<00:04,  8.12it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:07<00:04,  7.49it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:07<00:03,  8.90it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:08<00:03,  8.27it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:08<00:03,  7.43it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:08<00:02,  9.42it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:08<00:02,  9.03it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:02,  8.07it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:08<00:03,  7.62it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:09<00:02,  8.87it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:09<00:02,  8.71it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:09<00:01,  8.62it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:01,  8.46it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:10<00:01,  8.40it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:10<00:01,  7.92it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:10<00:01,  9.10it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:10<00:00,  8.83it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00, 10.71it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  9.66it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:11<00:00,  9.10it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  7.93it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.61it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  8%|###############                                                                                                                                                                             | 2/25 [00:00<00:01, 12.69it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:02,  9.41it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:02,  8.75it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:00<00:01, 11.01it/s] 40%|##########################################################################8                                                                                                                | 10/25 [00:00<00:01, 10.54it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:01<00:01, 11.25it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:01<00:00, 11.69it/s] 64%|#######################################################################################################################6                                                                   | 16/25 [00:01<00:00, 11.99it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:01<00:00,  9.41it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00,  8.91it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:02<00:00,  8.77it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  9.69it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  9.74it/s]
