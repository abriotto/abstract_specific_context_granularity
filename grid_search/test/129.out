# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6972959637641907, "acc": 0.5106281042098999, "length": 3.5543649196624756, "mode": "train", "epoch": 1}
{"loss": 0.6960315108299255, "acc": 0.5130114555358887, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6890128254890442, "acc": 0.5398571491241455, "length": 2.9680404663085938, "mode": "train", "epoch": 2}
{"loss": 0.683086097240448, "acc": 0.5441322326660156, "length": 1.4811784029006958, "mode": "test", "epoch": 2}
{"loss": 0.6466585993766785, "acc": 0.6593674421310425, "length": 3.2121787071228027, "mode": "train", "epoch": 3}
{"loss": 0.5867418050765991, "acc": 0.7342009544372559, "length": 3.89034366607666, "mode": "test", "epoch": 3}
{"loss": 0.536306619644165, "acc": 0.7429730296134949, "length": 3.854741096496582, "mode": "train", "epoch": 4}
{"loss": 0.5136802792549133, "acc": 0.7638298869132996, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.4655376970767975, "acc": 0.7927942872047424, "length": 3.954866886138916, "mode": "train", "epoch": 5}
{"loss": 0.4554218351840973, "acc": 0.8024548888206482, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.41314512491226196, "acc": 0.8245192170143127, "length": 3.9870615005493164, "mode": "train", "epoch": 6}
{"loss": 0.42800483107566833, "acc": 0.816857635974884, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.3860807716846466, "acc": 0.8416863679885864, "length": 3.98933744430542, "mode": "train", "epoch": 7}
{"loss": 0.4022713303565979, "acc": 0.8427168726921082, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.36670494079589844, "acc": 0.851166844367981, "length": 3.9943485260009766, "mode": "train", "epoch": 8}
{"loss": 0.3845597207546234, "acc": 0.8417349457740784, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.3512192666530609, "acc": 0.8605229258537292, "length": 3.995567560195923, "mode": "train", "epoch": 9}
{"loss": 0.37443771958351135, "acc": 0.8375613689422607, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.3386346995830536, "acc": 0.8640257716178894, "length": 3.9941630363464355, "mode": "train", "epoch": 10}
{"loss": 0.3683154881000519, "acc": 0.8538461923599243, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.3219051659107208, "acc": 0.8741732239723206, "length": 3.9978392124176025, "mode": "train", "epoch": 11}
{"loss": 0.3415321409702301, "acc": 0.8635843396186829, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.30486294627189636, "acc": 0.8801262974739075, "length": 3.9969277381896973, "mode": "train", "epoch": 12}
{"loss": 0.3238057792186737, "acc": 0.8730770349502563, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.2928966283798218, "acc": 0.888258695602417, "length": 3.9972145557403564, "mode": "train", "epoch": 13}
{"loss": 0.303356796503067, "acc": 0.8873975276947021, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.27819469571113586, "acc": 0.8931152820587158, "length": 3.996680736541748, "mode": "train", "epoch": 14}
{"loss": 0.30789777636528015, "acc": 0.8783140778541565, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.27384859323501587, "acc": 0.9002792835235596, "length": 3.9977293014526367, "mode": "train", "epoch": 15}
{"loss": 0.28454336524009705, "acc": 0.899427056312561, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.26165828108787537, "acc": 0.9054310917854309, "length": 3.9984569549560547, "mode": "train", "epoch": 16}
{"loss": 0.28147774934768677, "acc": 0.8995908498764038, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.25497013330459595, "acc": 0.9106903076171875, "length": 3.998114585876465, "mode": "train", "epoch": 17}
{"loss": 0.26021549105644226, "acc": 0.908674418926239, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.24975764751434326, "acc": 0.9106067419052124, "length": 3.9987120628356934, "mode": "train", "epoch": 18}
{"loss": 0.2658514678478241, "acc": 0.9067103862762451, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.24613064527511597, "acc": 0.911354124546051, "length": 3.998828172683716, "mode": "train", "epoch": 19}
{"loss": 0.25516876578330994, "acc": 0.9080195426940918, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.23852813243865967, "acc": 0.9153605103492737, "length": 3.999009132385254, "mode": "train", "epoch": 20}
{"loss": 0.24824205040931702, "acc": 0.9119476675987244, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.23551951348781586, "acc": 0.9178631901741028, "length": 3.999582290649414, "mode": "train", "epoch": 21}
{"loss": 0.24123691022396088, "acc": 0.914648175239563, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.22237558662891388, "acc": 0.9194982051849365, "length": 3.9984993934631348, "mode": "train", "epoch": 22}
{"loss": 0.23831582069396973, "acc": 0.9166122078895569, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.2197331339120865, "acc": 0.9226939082145691, "length": 3.999282121658325, "mode": "train", "epoch": 23}
{"loss": 0.24354134500026703, "acc": 0.9125203490257263, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.21667234599590302, "acc": 0.9218282103538513, "length": 3.9989864826202393, "mode": "train", "epoch": 24}
{"loss": 0.24201945960521698, "acc": 0.9169394373893738, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.2118535190820694, "acc": 0.9236605763435364, "length": 3.9996931552886963, "mode": "train", "epoch": 25}
{"loss": 0.2314554750919342, "acc": 0.914075493812561, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.20948374271392822, "acc": 0.9254675507545471, "length": 3.9997432231903076, "mode": "train", "epoch": 26}
{"loss": 0.24165354669094086, "acc": 0.9143208265304565, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.20529136061668396, "acc": 0.9248626232147217, "length": 3.999558210372925, "mode": "train", "epoch": 27}
{"loss": 0.22284936904907227, "acc": 0.9200490117073059, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.1953907459974289, "acc": 0.9313207864761353, "length": 3.9997828006744385, "mode": "train", "epoch": 28}
{"loss": 0.2177250236272812, "acc": 0.9230769276618958, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.1941835731267929, "acc": 0.9335553050041199, "length": 3.999117612838745, "mode": "train", "epoch": 29}
{"loss": 0.22244468331336975, "acc": 0.919476330280304, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.19919942319393158, "acc": 0.9272652864456177, "length": 3.9996817111968994, "mode": "train", "epoch": 30}
{"loss": 0.23232750594615936, "acc": 0.9171848893165588, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.19584870338439941, "acc": 0.9279597997665405, "length": 3.9998302459716797, "mode": "train", "epoch": 31}
{"loss": 0.23217444121837616, "acc": 0.9205400943756104, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.19513262808322906, "acc": 0.9281752109527588, "length": 3.9993393421173096, "mode": "train", "epoch": 32}
{"loss": 0.22448153793811798, "acc": 0.9244681596755981, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.19610203802585602, "acc": 0.9285065531730652, "length": 3.9998483657836914, "mode": "train", "epoch": 33}
{"loss": 0.21940970420837402, "acc": 0.9202128648757935, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.18790674209594727, "acc": 0.9298709034919739, "length": 3.9997167587280273, "mode": "train", "epoch": 34}
{"loss": 0.21266627311706543, "acc": 0.9234861731529236, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.175018310546875, "acc": 0.9353955984115601, "length": 3.9998745918273926, "mode": "train", "epoch": 35}
{"loss": 0.2105371057987213, "acc": 0.9229133129119873, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.17904908955097198, "acc": 0.9348782300949097, "length": 3.999910593032837, "mode": "train", "epoch": 36}
{"loss": 0.20911939442157745, "acc": 0.9252863526344299, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.18105840682983398, "acc": 0.9334027171134949, "length": 3.999786615371704, "mode": "train", "epoch": 37}
{"loss": 0.20400705933570862, "acc": 0.9274142384529114, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.1778784990310669, "acc": 0.9343863725662231, "length": 3.999919891357422, "mode": "train", "epoch": 38}
{"loss": 0.21950900554656982, "acc": 0.9240588545799255, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.1780025064945221, "acc": 0.9356167316436768, "length": 3.999929428100586, "mode": "train", "epoch": 39}
{"loss": 0.2081301063299179, "acc": 0.9220949411392212, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.16530367732048035, "acc": 0.9376941323280334, "length": 3.9999001026153564, "mode": "train", "epoch": 40}
{"loss": 0.19438432157039642, "acc": 0.9328151941299438, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.15842624008655548, "acc": 0.9411932826042175, "length": 3.9998860359191895, "mode": "train", "epoch": 41}
{"loss": 0.2007359117269516, "acc": 0.9315875768661499, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.1580428183078766, "acc": 0.9419274926185608, "length": 3.9997518062591553, "mode": "train", "epoch": 42}
{"loss": 0.19195841252803802, "acc": 0.9328969717025757, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.159208282828331, "acc": 0.9409139156341553, "length": 3.999796152114868, "mode": "train", "epoch": 43}
{"loss": 0.1982039362192154, "acc": 0.9328151941299438, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.15475979447364807, "acc": 0.9433251619338989, "length": 3.9999070167541504, "mode": "train", "epoch": 44}
{"loss": 0.1910695880651474, "acc": 0.9349427223205566, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.15074190497398376, "acc": 0.9443327784538269, "length": 3.99981689453125, "mode": "train", "epoch": 45}
{"loss": 0.1932043582201004, "acc": 0.9374794363975525, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.15450957417488098, "acc": 0.9454307556152344, "length": 3.999903678894043, "mode": "train", "epoch": 46}
{"loss": 0.19636957347393036, "acc": 0.9319966435432434, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.15195253491401672, "acc": 0.944469153881073, "length": 3.9991416931152344, "mode": "train", "epoch": 47}
{"loss": 0.18061776459217072, "acc": 0.9408345818519592, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.14524132013320923, "acc": 0.9458437561988831, "length": 3.9999823570251465, "mode": "train", "epoch": 48}
{"loss": 0.18436568975448608, "acc": 0.9369885325431824, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.14322292804718018, "acc": 0.9484924674034119, "length": 3.9999072551727295, "mode": "train", "epoch": 49}
{"loss": 0.18177230656147003, "acc": 0.9398525953292847, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.14581188559532166, "acc": 0.9472326040267944, "length": 3.999910593032837, "mode": "train", "epoch": 50}
{"loss": 0.1776694804430008, "acc": 0.9445169568061829, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.14021414518356323, "acc": 0.948575496673584, "length": 3.999918222427368, "mode": "train", "epoch": 51}
{"loss": 0.17730045318603516, "acc": 0.9391161799430847, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.15009444952011108, "acc": 0.9456579089164734, "length": 3.9996182918548584, "mode": "train", "epoch": 52}
{"loss": 0.19799673557281494, "acc": 0.93486088514328, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.15362855792045593, "acc": 0.9436005353927612, "length": 3.9999351501464844, "mode": "train", "epoch": 53}
{"loss": 0.1867053508758545, "acc": 0.9382976293563843, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.14517495036125183, "acc": 0.9463334083557129, "length": 3.9993197917938232, "mode": "train", "epoch": 54}
{"loss": 0.18326127529144287, "acc": 0.9428805112838745, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.14444679021835327, "acc": 0.9476887583732605, "length": 3.9998114109039307, "mode": "train", "epoch": 55}
{"loss": 0.18608008325099945, "acc": 0.9375615119934082, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.1493234932422638, "acc": 0.9461179375648499, "length": 3.99999737739563, "mode": "train", "epoch": 56}
{"loss": 0.1905573308467865, "acc": 0.9427985548973083, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.14761380851268768, "acc": 0.9457305073738098, "length": 3.999880313873291, "mode": "train", "epoch": 57}
{"loss": 0.17406730353832245, "acc": 0.9399346113204956, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.14739586412906647, "acc": 0.9496169090270996, "length": 3.999992609024048, "mode": "train", "epoch": 58}
{"loss": 0.17328038811683655, "acc": 0.9382979273796082, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.13725119829177856, "acc": 0.9502723217010498, "length": 3.999976396560669, "mode": "train", "epoch": 59}
{"loss": 0.17039695382118225, "acc": 0.9452535510063171, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.13251546025276184, "acc": 0.9527490735054016, "length": 3.9997708797454834, "mode": "train", "epoch": 60}
{"loss": 0.15284697711467743, "acc": 0.9486086964607239, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.13158054649829865, "acc": 0.9518582224845886, "length": 3.999993085861206, "mode": "train", "epoch": 61}
{"loss": 0.15882432460784912, "acc": 0.9450899362564087, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.1287393420934677, "acc": 0.9541550278663635, "length": 3.999995470046997, "mode": "train", "epoch": 62}
{"loss": 0.1545795500278473, "acc": 0.9515549540519714, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.12738972902297974, "acc": 0.9545642137527466, "length": 3.999983072280884, "mode": "train", "epoch": 63}
{"loss": 0.15648411214351654, "acc": 0.950900137424469, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.12381310015916824, "acc": 0.953991174697876, "length": 3.9999961853027344, "mode": "train", "epoch": 64}
{"loss": 0.1473875343799591, "acc": 0.9508182406425476, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.1270330399274826, "acc": 0.9544830322265625, "length": 3.9999961853027344, "mode": "train", "epoch": 65}
{"loss": 0.15398667752742767, "acc": 0.9511455297470093, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.12707418203353882, "acc": 0.9535800218582153, "length": 3.999988317489624, "mode": "train", "epoch": 66}
{"loss": 0.1495199054479599, "acc": 0.9482815265655518, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.12716782093048096, "acc": 0.9532254338264465, "length": 3.999995708465576, "mode": "train", "epoch": 67}
{"loss": 0.15984320640563965, "acc": 0.9438625574111938, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.1298195868730545, "acc": 0.9534451365470886, "length": 3.999432325363159, "mode": "train", "epoch": 68}
{"loss": 0.1814941018819809, "acc": 0.9468902349472046, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.1249135211110115, "acc": 0.9555219411849976, "length": 3.9999969005584717, "mode": "train", "epoch": 69}
{"loss": 0.17424315214157104, "acc": 0.9414893388748169, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.122225821018219, "acc": 0.9590756297111511, "length": 3.999997138977051, "mode": "train", "epoch": 70}
{"loss": 0.15981325507164001, "acc": 0.9515547752380371, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.11888695508241653, "acc": 0.9607435464859009, "length": 3.9999992847442627, "mode": "train", "epoch": 71}
{"loss": 0.15966416895389557, "acc": 0.9582650661468506, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.13422667980194092, "acc": 0.9592398405075073, "length": 3.999997854232788, "mode": "train", "epoch": 72}
{"loss": 0.17642240226268768, "acc": 0.9364157319068909, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.13184525072574615, "acc": 0.9512706398963928, "length": 3.9997775554656982, "mode": "train", "epoch": 73}
{"loss": 0.16479289531707764, "acc": 0.9409163594245911, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.12400604784488678, "acc": 0.9577223062515259, "length": 3.9998104572296143, "mode": "train", "epoch": 74}
{"loss": 0.13973093032836914, "acc": 0.956301212310791, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.11653000116348267, "acc": 0.9600890874862671, "length": 3.9997100830078125, "mode": "train", "epoch": 75}
{"loss": 0.1500941812992096, "acc": 0.9462355375289917, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.1181037425994873, "acc": 0.9576213359832764, "length": 3.999885320663452, "mode": "train", "epoch": 76}
{"loss": 0.14451657235622406, "acc": 0.9576104283332825, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.10568477213382721, "acc": 0.964952826499939, "length": 3.9999918937683105, "mode": "train", "epoch": 77}
{"loss": 0.14830707013607025, "acc": 0.9545827507972717, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.1044468954205513, "acc": 0.9639955163002014, "length": 3.9999876022338867, "mode": "train", "epoch": 78}
{"loss": 0.14500939846038818, "acc": 0.9560555815696716, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.10507399588823318, "acc": 0.9644299745559692, "length": 3.999941110610962, "mode": "train", "epoch": 79}
{"loss": 0.14948886632919312, "acc": 0.9583468437194824, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.1075163185596466, "acc": 0.9645160436630249, "length": 3.999999523162842, "mode": "train", "epoch": 80}
{"loss": 0.1929781287908554, "acc": 0.9496727585792542, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.12746475636959076, "acc": 0.9600865840911865, "length": 3.9999897480010986, "mode": "train", "epoch": 81}
{"loss": 0.15435676276683807, "acc": 0.9563830494880676, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.1093054935336113, "acc": 0.9618083238601685, "length": 3.99998140335083, "mode": "train", "epoch": 82}
{"loss": 0.14064744114875793, "acc": 0.9579378962516785, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.10692406445741653, "acc": 0.9646477103233337, "length": 3.9991614818573, "mode": "train", "epoch": 83}
{"loss": 0.14871850609779358, "acc": 0.9600655436515808, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.10223150998353958, "acc": 0.9661290049552917, "length": 3.999999761581421, "mode": "train", "epoch": 84}
{"loss": 0.13582468032836914, "acc": 0.9599835276603699, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.1057974174618721, "acc": 0.9651448130607605, "length": 3.999999523162842, "mode": "train", "epoch": 85}
{"loss": 0.13985702395439148, "acc": 0.9601472616195679, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.10582656413316727, "acc": 0.964953601360321, "length": 4.0, "mode": "train", "epoch": 86}
{"loss": 0.12751851975917816, "acc": 0.9608020186424255, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.10752305388450623, "acc": 0.9619424343109131, "length": 3.999917507171631, "mode": "train", "epoch": 87}
{"loss": 0.13143567740917206, "acc": 0.9566285014152527, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.10666383802890778, "acc": 0.9636685848236084, "length": 4.0, "mode": "train", "epoch": 88}
{"loss": 0.13830669224262238, "acc": 0.9605566263198853, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.10532400012016296, "acc": 0.9660469889640808, "length": 4.0, "mode": "train", "epoch": 89}
{"loss": 0.1482314169406891, "acc": 0.9527824521064758, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.11375143378973007, "acc": 0.9614269137382507, "length": 3.999999761581421, "mode": "train", "epoch": 90}
{"loss": 0.150691419839859, "acc": 0.9574466943740845, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.10795728862285614, "acc": 0.9655002355575562, "length": 3.9999992847442627, "mode": "train", "epoch": 91}
{"loss": 0.13800135254859924, "acc": 0.9602292776107788, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.1000061184167862, "acc": 0.966648519039154, "length": 3.999999761581421, "mode": "train", "epoch": 92}
{"loss": 0.14353330433368683, "acc": 0.9600653052330017, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.09567227959632874, "acc": 0.9671679735183716, "length": 4.0, "mode": "train", "epoch": 93}
{"loss": 0.13131164014339447, "acc": 0.9618658423423767, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.09297080338001251, "acc": 0.9681519269943237, "length": 4.0, "mode": "train", "epoch": 94}
{"loss": 0.1293146163225174, "acc": 0.9554010033607483, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.09445609152317047, "acc": 0.9679332375526428, "length": 3.9999992847442627, "mode": "train", "epoch": 95}
{"loss": 0.11698619276285172, "acc": 0.9641571044921875, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.08902756869792938, "acc": 0.9702023863792419, "length": 3.999999761581421, "mode": "train", "epoch": 96}
{"loss": 0.12903490662574768, "acc": 0.9649754166603088, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.08855752646923065, "acc": 0.9709676504135132, "length": 4.0, "mode": "train", "epoch": 97}
{"loss": 0.12118683010339737, "acc": 0.9651390314102173, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.0859132707118988, "acc": 0.9701475501060486, "length": 3.9999990463256836, "mode": "train", "epoch": 98}
{"loss": 0.12412961572408676, "acc": 0.9648938179016113, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.08175867050886154, "acc": 0.9716237783432007, "length": 3.999999523162842, "mode": "train", "epoch": 99}
{"loss": 0.13201498985290527, "acc": 0.9645662307739258, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.08071506768465042, "acc": 0.9724165201187134, "length": 3.999999523162842, "mode": "train", "epoch": 100}
{"loss": 0.12129884213209152, "acc": 0.9661211967468262, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.6709820628166199
